packages/client/public/index.html

```diff
@@ -10,34 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
```

Comment:
This was generated by React.js itself, so no need to modify this file 😄

Reply:
You're right! I have deleted some files in React.js, because it's filled with irrelevant contents. And we used to remove in our class as well. That's the reason behind!